components/eam/src/dynamics/se/inidat.F90

```diff
@@ -424,7 +443,8 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
                 indx = indx + 1
              end do
           end do
-       end do
+ 
```

Comment:
No need for these changes, please revert to clean up the diffs.

Reply:
Fixed. The indentation is also not consistent through this entire section of the file; since much of it is code not touched by this commit, however, I'm leaving it as it.